tests/console/test_Log.py

```diff
@@ -1,28 +1,44 @@
 import datetime
-from pathlib import Path
-from unittest.mock import MagicMock
+from io import TextIOBase
+from unittest.mock import MagicMock, PropertyMock
 
 import pytest
 from rich.traceback import Trace
 
 import briefcase
-from briefcase.console import Log
+from briefcase.commands.dev import DevCommand
+from briefcase.console import Console, Log
 from briefcase.exceptions import BriefcaseError
 
 TRACEBACK_HEADER = "Traceback (most recent call last)"
 EXTRA_HEADER = "Extra information:"
 
 
-@pytest.fixture
+@pytest.fixture(autouse=True)
```

Comment:
I'm not a big fan of `autouse` - outside of "whole of test suite" configuration setup, I find explicit is better than implicit. For the sake of including a `now` fixture in a handful of tests, which makes it clear that "this test has a specific time interpretation".

Reply:
That's fair; also removed it from Windows SDK tests.